tasks/artifact.xml

```diff
@@ -0,0 +1,314 @@
+<?xml version="1.0"?>
+
+<!--
+  @file artifact.xml
+  Targets for managing Drupal code artifacts.
+
+  Copyright 2018 Palantir.net, Inc.
+  -->
+
+<project name="artifact" default="artifact-main">
+
+
+    <!--
+      Required configuration for building artifacts:
+        artifact.directory
+        artifact.git.remote
```

Comment:
Should this have a description?  It wasn't initially clear to me that it was the full URL to the Git repository on the host.